Update rules for RHEL 9 STIG

jan-cerny opened the pull request:
#### Description:
According to feedback from DISA we updated some rules and their OCIL, fixtext, SRG requirements, etc.

#### Rationale:
Update the rules for RHEL 9 STIG.

vojtapolasek commented on controls/srg_gpos.yml:
```diff
@@ -27,5 +27,6 @@ controls:
             - var_account_disable_post_pw_expiration=35
             - var_auditd_action_mail_acct=root
             - var_auditd_space_left_percentage=25pc
+            - var_auditd_admin_space_left_percentage=5pc
```
Why not to ad this to the relevant srg_gpos file in the subdirectory? It would be closer to the related rule. Or are you following some coding rule?

jan-cerny commented:
I have moved variable setting to SRG.

vojtapolasek commented on linux_os/guide/system/software/updating/ensure_gpgcheck_globally_activated/rule.yml:
```diff
@@ -86,8 +86,11 @@ platform: yum
 srg_requirement: |-
-   {{{ full_name }}} must have GPG checking enabled in the {{{ pkg_manager }}} package manager.
+    {{{ full_name }}} must prevent the installation of software, patches, service packs, device drivers, or operating system components from a repository without verification they have been digitally signed using a certificate that is issued by a Certificate Authority (CA) that is recognized and approved by the organization.
+    {{{ full_name }}} must have GPG checking enabled in the {{{ pkg_manager }}} package manager.
```
Are you sure that srg_requirement should have two entries like this?

jan-cerny replied:
it shouldn't be there.

vojtapolasek commented on linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/rule.yml:
```diff
@@ -46,20 +46,24 @@ references:
 ocil: |-
-    To verify that <tt>localpkg_gpgcheck</tt> is configured properly, run the following
-    command:
+    Verify that dnf verifies the signature of local packages prior to install with the following command:
```
I think DNF can be replaced by the Jinja variable here.

jan-cerny replied:
yes

vojtapolasek reviewed (changes requested):
Thank you for this update. Please see my comments.

jan-cerny commented on linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_command/rule.yml:
```diff
@@ -33,30 +33,20 @@ references:
 
     <pre>set -g lock-command vlock</pre>
 
-    Then, verify that the /etc/tmux.conf file can be read by other users than root:
```
We have add the file permission checks to the tmux rules by https://github.com/ComplianceAsCode/content/pull/9080 as a solution for https://bugzilla.redhat.com/show_bug.cgi?id=2064696. Back then, we decided (discussion in the PR) to put the file permissions into the rule because they are tightly coupled with the actual config option and it's hard to track dependencies among rules.

Now I don't know, we can have a special field with a check text just for STIG, but then it could be seen as misaligned content.

jan-cerny replied:
The problem will be tracked in https://github.com/ComplianceAsCode/content/issues/9533

jan-cerny commented on linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_data_retention_admin_space_left_percentage/ansible/shared.yml:
```diff
@@ -0,0 +1,15 @@
+    regexp: '^\s*admin_space_left\s*=\s*.*$'
+    state: present
+    create: yes
+  #notify: reload auditd
```
It's a copy paste problem.

jan-cerny commented:
I have removed the comment from the Ansible remediation, replaced dnf by a variable, removed the sententce from SRG requirement field. Then, I have removed the removal of text related to permissions of the tmux config file, because we need to solve that separately.

vojtapolasek commented:
LGTM. Thank you.
